### PR TITLE
CFD-478 : featured block is moving around the pictures in the container generating white space

### PR DESCRIPTION
**Pull request by solker**
https://issuetracker.amplexor.com/jira/browse/CFD-478

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@solker Add screenshot, please. After tests are green - assign Peter and move issue in Jira to the quality control.

**Comment by solker** [comment]
@HelenaEksler 

selection_010

**Comment by solker** [comment]
@HelenaEksler ready for code review.

* The block title removed.
* New style image created -> https://wiki.amplexor.com/confluence/display/C4DNG/Design+standards
* Image style applied to all images in Featured block.
* Set trim text (200 characters - 2 line) option for body text.
* Set padding like in wireframe -> https://wiki.amplexor.com/confluence/display/C4DNG/C4D+PC+Featured
selection_017

**Comment by HelenaEksler** [comment]
@solker Looks good. Assign to Peter or Marteen as soon as tests are green.

File: capacity4more/modules/c4m/content/c4m_content_featured/plugins/content_types/featured_block/featured_block.inc [hunk 1]
```diff
@@ -19,7 +19,6 @@ $plugin = array(
 function c4m_content_featured_featured_block_content_type_render($subtype, $conf, $args, $context) {
   $block = new stdClass();
   $block->module = 'c4m_content_featured';
-  $block->title = t('Featured block');
```
Comment (solker) [inline capacity4more/modules/c4m/content/c4m_content_featured/plugins/content_types/featured_block/featured_block.inc]: Or better `$block->title = '';` ?

Comment (HelenaEksler) [inline capacity4more/modules/c4m/content/c4m_content_featured/plugins/content_types/featured_block/featured_block.inc]: Yeah, Make ``$block->title = '';``

**Comment by HelenaEksler** [comment]
@solker Please, check why tests are not passing.

**Comment by solker** [comment]
@HelenaEksler Behat test fixed.

File: capacity4more/behat/features/homepage.feature [hunk 2]
```diff
@@ -79,11 +79,11 @@ Feature: Test homepage content and blocks
     Then  I should see "Upcoming events" in the "div.pane-upcoming-events" element
     And   I should see only "2" events
 
-  @api
+  @api @test
```
Comment (HelenaEksler) [inline capacity4more/behat/features/homepage.feature]: remove ``@test``

**Comment by MPParsley** [comment]
@solker, can you make the block accessible to mobile devices?

featured mobile small
featured mobile medium

Probably best to set images width to 100% and height to auto.

**Comment by solker** [comment]
@MPParsley you are welcome.
I made gif grey because this file so big and here limit to 10 MB.
478-responsible 1

**Pull request merged**